### ISSUE TITLE
Problem in DNS script 0.5.1

### ISSUE THREAD

**Issue by dapolinario** [Chadsr/NordVPN-NetworkManager#90]
In this version 0.5.1, when the custom DNS option is not set, the generated DNS script does not receive the DNS addresses returned by the get_nameservers function.

Another point, when the VPN connection occurs, DNS addresses are sent via DHCP and updated in /etc/resolv.conf, even then DNS leak protection is required?

**Comment by dapolinario** [comment]
Clarifying: I think the script is unnecessary.

This is my `/etc/resolv.conf` before the script `/etc/NetworkManager/dispatcher.d/nordnm_dns_USER` runs and after the VPN connect (receiving information via DHCP, including DNS):

```
# Generated by NetworkManager
search BLABLABLA.COM
nameserver 103.86.96.100
nameserver 103.86.99.100
nameserver 192.168.2.1
```

The script would only be needed for customizable DNS or to remove my DNS (192.168.2.1), which may actually be something valid because it may cause leak.

**Comment by dapolinario** [comment]
I did a search and I deem the script necessary only for using custom DNS. I tested the following procedure and it worked perfectly: https://wiki.archlinux.org/index.php/OpenVPN#Override_DNS_servers_using_NetworkManager

**Comment by redni** [comment]
I was supposed to create an ISSUE on an empty resolv.conf, but this should probably be one of those too. After typing `sudo nordnm sync -uka pl p2p udp` when you do not specify your own DNS during configuration, the resolv conf file contains a line without IP

`nameserver ` instead of  `namserver IP`

breaks down the whole internet and additionally the rights of the file resolv.conf are set so that it can not be deleted or edited without special commands.

**Comment by Chadsr** [comment]
@dapolinario thanks for pointing out that networkmanager flag - I had somehow not come across its existence. I will test the changes in your PR (#91) and see if I agree that the script is not necessary with these changes (but it definitely looks like it isn't).

@redni Very sorry for this screw-up. I already have a fix for this pushed (bc15cc0) and will be releasing it shortly. If PR #91 seems like a good solution, I will also be completely removing the need for nordnm manipulating resolv.conf directly. So no more immutable flag and potential fuck-ups there.

**Comment by Chadsr** [comment]
Well, the fixes are ready to release, but Travis CI is having some problems and is preventing me from actually releasing anything right now... (https://www.traviscistatus.com/incidents/0ch1w22m3lwc)

I will have the update rolled out as soon as they sort there stuff out.

**Comment by Chadsr** [comment]
This should now be fixed in v0.5.3 onwards, so I will close the issue. If you find any more issues related to this topic feel free to re-open it!
